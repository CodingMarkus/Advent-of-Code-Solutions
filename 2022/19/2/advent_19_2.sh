#!/bin/sh

# Solution to https://adventofcode.com/2022/day/19

# Run as:
# cat advent_19_2_sample.txt | sh advent_19_2.sh
# cat advent_19_2_input.txt  | sh advent_19_2.sh

set -e
