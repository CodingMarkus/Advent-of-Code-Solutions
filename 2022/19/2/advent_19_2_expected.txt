Product: 3472
Product: 2415
